lib/models/quellenreiter_app_state.dart

```diff
@@ -257,8 +257,8 @@ class QuellenreiterAppState extends ChangeNotifier {
   /// Callback for the signup process. If successfull, a [Player] is returned and safed.
   /// If not, the route is set to signup.
   ///
-  /// @param p The [Player] returned from the server.
-  void _signUpCallback(Player? p) {
+  /// @param p The [LocalPlayer] returned from the server.
```

Comment:
Can we take these changes out of the commit? Does not seem related to the integration test, but instead to the milestone https://github.com/QuellenReiter/quellenreiter_app/milestone/1, right?